docs/conf.py

```diff
@@ -72,12 +72,12 @@ def setup(sphinx):
 # The short X.Y version.
 with open('../CMakeLists.txt', 'r', encoding='utf8') as f:
     version = re.search('PROJECT_VERSION "([^"]+)"', f.read()).group(1)
-# The full version, including alpha/beta/rc tags.
-if not os.path.isfile('../prerelease.txt') or os.path.getsize('../prerelease.txt') == 0:
-    release = version
-else:
+if os.path.isfile('../cmake/prerelease.lock'):
     # This is a prerelease version
     release = version + '-develop'
```

Comment:
Is `-develop` always correct here? Wouldn't it make sense to use `SOL_VERSION_PRERELEASE`? Not sure whether this really makes sense here, but maybe CMake could write `SOL_VERSION_PRERELEASE` to a file somewhere, and this information is then just used here.

Reply:
Ah ok, this line was not touched. So maybe we don't want to change that.

Reply:
Yeah, I think it's fine as is. We don't need really that level of granularity for docs. I mean, would be nice to have the exact version there but I also think it's not worth complicating the build system.